site/source/tutorials/reactive_programming.md

```diff
@@ -0,0 +1,58 @@
+# Reactive Programming
+
+Reactive programming is concenered with handling change, specifically, the _propagation of change_. As web applications have become more and more asynchronous, keeping track of application state has become increasingly complicated. With traditional programming patterns we rely on event listeners or try to keep track of application state manually.
+
+Consider a banking application where the user's account balance might show up in several different places on the screen. After the user makes a withdrawl, we want the account balance to update.
+
+## Imperative Approach
+
+Traditionally, we might use an event emitter. Every time balance is updated we emit an event, and any interested parties can listen in.
```

Comment:
Every time balance is updated we emit an event, and any interested parties can listen in. -> Every time the balance is updated, we emit an event, and any interested parties can listen for these events.